ibm_mq/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # CHANGELOG - IBM MQ
 
+## 2.0.0 / 2019-04-16
+
+* [Changed] Change host for mq_host. See [#3608](https://github.com/DataDog/integrations-core/pull/3608).
```

Comment:
Can we make this a little more clear? This is the breaking change, customers should be able to easily tell whether it's impacting them.